components/content/ProseA.vue

```diff
@@ -29,5 +29,5 @@ const props = defineProps({
 
 const isApiReferenceLink = props.href.startsWith('https://docs.centrapay.com/api');
 
-const isExternalLink = !props.href.startsWith('https://docs.centrapay.com') || isApiReferenceLink;
+const isExternalLink = (!props.href.startsWith('/') && !props.href.startsWith('https://docs.centrapay.com')) || isApiReferenceLink;
```

Comment:
This allows for partial linking i.e. `/guides` for an internal link to run locally.